sklearn/tests/test_calibration.py

```diff
@@ -319,6 +342,18 @@ def test_calibration_less_classes():
         assert np.all(np.hstack([proba[:, :i],
                                  proba[:, i + 1:]]))
 
+    # When ensemble=False it should return uniform probabilities
+    cal_clf = CalibratedClassifierCV(clf, method="sigmoid", cv=LeaveOneOut(),
+                                     ensemble=False)
+    cal_clf.fit(X, y)
+
+    for i, calibrated_classifier in \
+            enumerate(cal_clf.calibrated_classifiers_):
+        proba = calibrated_classifier.predict_proba(X)
+        assert np.all(proba == proba[0, 0])
+        assert np.all(np.hstack([proba[:, :i],
+                                 proba[:, i + 1:]]))
```

Comment:
if you're checking for uniform proba, maybe just assert something like `np.allclose(proba, 1 / proba.shape[0])`

Reply:
ok